integrations/axum-garde/Cargo.toml

```
@@ -0,0 +1,47 @@
+[package]
+name = "axum-garde"
+version = "0.1.0"
+edition = "2021"
+
```

Yes, sorry for the confusion. It needs a license and a few other fields:
```suggestion
repository = "https://github.com/jprochazk/garde"
license = "MIT OR Apache-2.0"
description = "Axum integration for garde"
```